apps/src/blockly/addons/functionEditor.ts

```diff
@@ -146,20 +150,21 @@ export default class FunctionEditor {
   }
 
   getWorkspaceId() {
-    return this.editorWorkspace.id;
+    return this.editorWorkspace?.id;
   }
 
   getWorkspace() {
     return this.editorWorkspace;
   }
 
-  // TODO
-  renameParameter(oldName, newName) {}
+  // Leaving these two functions as placeholders for when we implement parameters.
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
+  renameParameter(_oldName: string, _newName: string) {}
 
-  // TODO
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
   refreshParamsEverywhere() {}
```

Comment:
Just noting that `refreshParamsEverywhere` and `renameParameter` are supposedly used in Dance by `sprite_parameter_get` blocks, but I don't think those even exist in that lab. It would be great for us to clean up some of the dance block logic that never actually got used, because then we could remove some of these methods (from here and the wrapper).

Reply:
yeah I left these for now because they are theoretically used, but I'm open to removing these as a follow up!